### ISSUE TITLE
Fieldset incorrectly gets set in error when a child input is in error

### ISSUE THREAD

**Issue by edwardhorsford** [NHSDigital/nhsuk-react-components#245]
It seems like the fieldset component will set itself in error if one of the child inputs inside it is in error. This is not correct - it should only be in error if the fieldset as a whole is in error.

The result is that the left red error bar extends across the entire fieldset, rather than just the field that is in error.

You could possibly say that this is something to be decided on a question by question basis - but then this shouldn't be the default as it's not how the native components work.

It's not in the docs, but reading the package source, I see you can use `<Fieldset disableErrorLine="false">` to explicitly disable this.

Incorrect:
![Screenshot 2024-07-15 at 13 46 35](https://github.com/user-attachments/assets/40a4faaa-6516-48fa-aa87-bfead58cc708)

Correct:
![Screenshot 2024-07-15 at 13 58 07](https://github.com/user-attachments/assets/21b2ad95-2f35-49a8-8858-4692e2ac5732)

There's no example of this in the NHS design system, but it's covered in the [GOV.UK design system](https://design-system.service.gov.uk/patterns/addresses/#error-messages):
![Screenshot 2024-07-15 at 13 58 58](https://github.com/user-attachments/assets/83075334-cf02-4f14-9d60-afa8dad2cd98)

**Comment by colinrotherham** [comment]
@edwardhorsford @rowellx68 Can you see if I've fixed this in https://github.com/NHSDigital/nhsuk-react-components/pull/267?

**Comment by colinrotherham** [comment]
Yeah here's the [**Upgrading to 6.0** guide](https://github.com/NHSDigital/nhsuk-react-components/blob/nhsuk-frontend-10.0.0/docs/upgrade-to-6.0.md)

Fieldsets are no longer necessary in the sense that, if a component wants one, it does it itself

**Comment by edwardhorsford** [comment]
I don't see `disableErrorLine` being mentioned in the guide - should we mention that it's been removed?

**Comment by colinrotherham** [comment]
> I don't see disableErrorLine being mentioned in the guide - should we mention that it's been removed?

Yeah, for people that forget to remove `<Fieldset>`?

It's still there on every form component and the ~`<SingleInputFormGroup>`~ `<FormGroup>` wrapper
